### ISSUE TITLE
Error to run the tflite model: self._interpreter.Invoke() RuntimeError: Input tensor 1283 lacks data

### ISSUE THREAD

**Issue by JiashuGuo** [tensorflow/tensorflow#51404]
### 1. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux 18.04
- TensorFlow installation (pip package or built from source): pip
- TensorFlow library (version, if pip package or github SHA, if built from source): 2.7.0-dev20210804

### 2. Code
The code I use to convert the model and run the interpreter:

```
converter = tf.lite.TFLiteConverter.from_saved_model("mask_rcnn_coco_1204")
converter.optimizations = [tf.lite.Optimize.DEFAULT]
converter.representative_dataset = representative_data_gen
converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8, tf.lite.OpsSet.SELECT_TF_OPS]
converter.inference_input_type = tf.uint8
converter.inference_output_type = tf.uint8
converter.experimental_new_converter = True
tflite_model = converter.convert()
fo = open(
    "mask_rcnn_inception_1024_int_quantized.tflite", "wb")
fo.write(tflite_model)
fo.close

interpreter = tf.lite.Interpreter(model_path="mask_rcnn_inception_1024_int_quantized.tflite")
interpreter.allocate_tensors()
input_details = interpreter.get_input_details()
output_details = interpreter.get_output_details()

input_shape = input_details[0]['shape']
input_data = np.array(np.random.random_sample(input_shape), dtype=np.uint8)
interpreter.set_tensor(input_details[0]['index'], input_data)

interpreter.invoke()

output_data = interpreter.get_tensor(output_details[0]['index'])
print(output_data)
```
Here is the [tflite model](https://drive.google.com/file/d/1yvvxl9iq2CpFqZ7_a4LMKZkpTppRJdGV/view?usp=sharing) that converted. 
### 3. Failure after conversion
Model converting finished with no error, but the converted model couldn't run through the interpreter. 

### 4. Error traceback
`Traceback (most recent call last):
  File "converter.py", line 100, in <module>
    interpreter.invoke()
  File "/home/dev/.local/lib/python3.6/site-packages/tensorflow/lite/python/interpreter.py", line 858, in invoke
    self._interpreter.Invoke()
RuntimeError: Input tensor 1283 lacks data
`

_Originally posted by @JiashuGuo in https://github.com/tensorflow/tensorflow/issues/51209#issuecomment-895023730_

**Comment by sushreebarsa** [comment]
@JiashuGuo Thank you for the post! 
Can you please fill the issue [template](https://github.com/tensorflow/tensorflow/issues/new/choose) In order to expedite the trouble-shooting process of  the issue reported here. Thanks!

**Comment by JiashuGuo** [comment]
> @JiashuGuo Thank you for the post!
> Can you please fill the issue [template](https://github.com/tensorflow/tensorflow/issues/new/choose) In order to expedite the trouble-shooting process of the issue reported here. Thanks!

I updated the issue.

**Comment by srjoglekar246** [comment]
@abattery Do models with Flex ops work as-is in Python?

**Comment by abattery** [comment]
Yes, it works

**Comment by JiashuGuo** [comment]
> @abattery Do models with Flex ops work as-is in Python?

Have you found any fixes for this? Is it because of mask_rcnn model has operations that don't support quantization? I found the mask_rcnn model doesn't work with quantization-aware training either.

**Comment by srjoglekar246** [comment]
@JiashuGuo At a high level, may I know what is the output you desire from the MaskRCNN model? Is it just bounding boxes or segmentation masks as well?

Some of the TF MaskRCNN versions aren't mobile friendly due to their source graph, which generates a lot of problems especially if you are considering quantization.

**Comment by JiashuGuo** [comment]
I am expecting both bboxes and segment masks.

**Comment by srjoglekar246** [comment]
I see. Where was this Mask RCNN model obtained from? I am not very familiar with the model structure and what kinds of I/O tensors it expects. Maybe try running a floating point model before trying quantized?

**Comment by JiashuGuo** [comment]
> I see. Where was this Mask RCNN model obtained from? I am not very familiar with the model structure and what kinds of I/O tensors it expects. Maybe try running a floating point model before trying quantized?

The model is downloaded from [here](https://tfhub.dev/tensorflow/mask_rcnn/inception_resnet_v2_1024x1024/1).  I also tried other versions in tf1 model zoo, but none works. So yeah, it seems now I can only run the float point model.

**Comment by srjoglekar246** [comment]
To clarify: Does the floating point model work as expected?

**Comment by JiashuGuo** [comment]
> To clarify: Does the floating point model work as expected?

Yes, it works.

**Comment by pjpratik** [comment]
Hi @JiashuGuo 

I was checking if the issue still exists.

The issue seems to be resolved with TF 2.12. I was able to invoke the tensor and print the output data with out any error.

Please refer this [gist](https://colab.research.google.com/gist/pjpratik/f4be6b438d820468529322f784316451/51404.ipynb) and let us know if it helps.

Thanks.
